### ISSUE TITLE
Namespace the ops

### ISSUE THREAD

**Issue by bbatsov** [clojure-emacs/cider-nrepl#710]
Right now `cider-nrepl` doesn't namespace its ops, which in the past became problematic when we decided to add a completion op to nREPL. Normally it would have been named `complete`, but we had to go with a different name, as `cider-nrepl` already had `complete`.

It'd be nice to give all the ops names like `cider/info`, `cider/complete`, etc. This has to happen in several steps, though:

* first we need to add the new names alongside the old names
* update the docs to promote the newer ops (and maybe send some notifications to clients when using the legacy names)
* give the clients a few years to update
* drop the old "global" names (optional step)

In general new ops shouldn't use the global namespace.  The same applies to `refactor-nrepl`.

**Comment by bbatsov** [comment]
Yeah, I think that makes sense, as we can have a collision in the name of responses as well. As requests and responses are normally matched on their IDs that's not a big problem for most clients, but the IDs are optional so who knows what someone might be using. 

I definitely regret not thinking more about the proper namespacing of ops earlier in the dev process. :D

**Comment by r0man** [comment]
Ok, I will prefix the responses as well then.
